Devfile_Cert_Criteria.md

```diff
@@ -0,0 +1,84 @@
+# Certified Stack Hub
+
+This document describes the criteria a Devfile application stack must implement to be considered a certified application stack.
+
+## Certification Criteria
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in [BCP
+14](https://tools.ietf.org/html/bcp14)
+[[RFC2119](https://tools.ietf.org/html/rfc2119)]
+[[RFC8174](https://tools.ietf.org/html/rfc8174)] when, and only when, they
+appear in all capitals, as shown here.
+
+### Stack structure
+
+* A certified Application Stack MUST also be an [Application Stack](https://github.com/IBM/cp4apps-hub/tree/devfile_cert#application-stacks), having the necessary files needed to fulfill core scenarios by the architect (Champ) and the developer (Jane).
+
+* Stacks MUST have a deployment manifest defined in the devfile, using the `alpha.deployment-manifest` property ,that specifies the deployment of its application runtime container, using either:
```

Comment:
I believe we should list the dockerfile too. Without a build there will be nothing to deploy.

Reply:
This can be added, what is the name of the property that indexes the dockerfile?

Reply:
alpha.build-dockerfile: <URI>
    alpha.deployment-manifest: <URI>

Reply:
Other than the `odo deploy` scenario, what about the `odo pipeline` scenario? Should we also add the requirement for pipeline in here as well (I don't know the details on the setup/config needed for that)?

Reply:
@elsony Is odo pipeline part of the 2.0 devfile specification? I am thinking we should limit this document to 2.0 and then it can be updated later for devfile 2.1. Does that sound reasonable?

Reply:
yep, I think that's fine.  Given that the development of pipeline part has been stopped for the time being (with the recent change in direction), the design may change anyways.